gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -138,12 +138,37 @@ fluid.defaults("gpii.flowManager.local", {
             type: "gpii.flowManager.getGpiiKey.handler"
         }
     },
+    invokers: {
+        // Needs to be implemented if the handling is required when users update their preferences.
+        setSettings: "fluid.identity"
```

Comment:
fluid.notImplemented is safer, plus an explicit implementation in the other derived class - which can probably just always return a promise

Reply:
The problem I encountered with using `fluid.notImplemented` is, this function reports [a failure](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/Fluid.js#L418) when the associated invoker is not implemented, which makes sense when this function presents as a placeholder/reminder. However when GPII runs in all-in-local config where the other derived "untrusted" class is not included, `fluid.notImplemented` will be executed and cease the run with a failure.

Let me know if I didn't use function in a proper way. Thanks.

Reply:
I've improved this placeholder function to return an empty promise as you suggested in the comment below.